### ISSUE TITLE
Sharing Bricks via libraries

### ISSUE THREAD

**Issue by s-steddy** [Scholar-6/brillder#4688]
xyz

**Comment by ivanromakh** [comment]
its possible to hide other options for library users probably.

**Comment by ivanromakh** [comment]
@s-steddy  So I need to leave only library login option in this popup?
![image](https://user-images.githubusercontent.com/18567511/187448239-1463589b-6355-4dbd-9c26-1b5e87ff7543.png)

**Comment by ivanromakh** [comment]
Its clear I will be working on this one.

**Comment by ivanromakh** [comment]
I will put on dev login with library barcode and pin probably for testing. almost done.

**Comment by s-steddy** [comment]
Looking great, a few more tweaks though:

At the top of the page:
- it looks like login|register (top right) leads to barcode/pin entry - excellent
- make sure the home button and search bar can't be used to navigate away, load the popup if user clicks on these
- Also stop the #tags from being clickable, but no need to load popup for these 

Popup Message:
**Wellcome to Brillder for libraries.**
Great that you've clicked a brick! 
To play for free, please connect to your library.

Buttons:
- First button design is fine, but let's say "Connect to your library"
- THEN the redirect message, but let's add: "Requires library card barcode and pin. You will be returned to this page."
- Then second button: "Stay on page". I don't much like the icon, do we have anything else suitable?

Play button:
- I was able to start playing by clicking 'keep exploring' after the popup, then 'play'. Users **must** be logged in (through library) to play, so prevent this, and load the popup if they try this.
- I was also able to start playing by clicking 'play', then 'keep exploring' from the popup - it jumped straight to the next step. Same as above, no advancing to play until user is logged in.

**Comment by s-steddy** [comment]
Just realised, eventually, we'll need a flow for users with a non-library account accessing a library-created link. it'll need to give them the option to convert to library membership and explain differences etc.

But let's get everything else working first.

**Comment by ivanromakh** [comment]
I can put some arrow icon. not sure if there is icon related to staying on page. I used arrow-right
![image](https://user-images.githubusercontent.com/18567511/188852543-341eb0ec-5e53-418b-97ea-a7187dc8b76a.png)

**Comment by ivanromakh** [comment]
we use arrow icons everywhere in play to move next

**Comment by ivanromakh** [comment]
made it more centered.
![image](https://user-images.githubusercontent.com/18567511/188854009-262d6e70-24f4-443a-abda-311ad1e87c77.png)

**Comment by ivanromakh** [comment]
@s-steddy I am not sure about play. all users can play untill review then they need to login.
![image](https://user-images.githubusercontent.com/18567511/188854981-743172d5-b5ed-40ba-837d-8eaf363700e0.png)

**Comment by s-steddy** [comment]
Yes, I tried to explain to them last time we spoke that they might want to let people play until the review before going through login, but they seemed to really want to make sure people were signed up and logged in through the library before anything else. Can we do it and then be ready to change back?

**Comment by ivanromakh** [comment]
Ok I will make exception

**Comment by s-steddy** [comment]
How about we stop 'Stay on page' even being a button, and have it as linked text at the bottom of the popup, with an X?

![2022-09-07 11 31 24](https://user-images.githubusercontent.com/107257153/188857115-4f8276cc-be05-433d-b76c-3d9cb93d870c.jpg)

**Comment by ivanromakh** [comment]
sure I will make it

**Comment by s-steddy** [comment]
Cool, ok just had one more idea:

For the connect button, just have it say simply 'Connect' 
and on the hover-help button, say 'Requires a library card barcode and pin from a participating library'

**Comment by ivanromakh** [comment]
@s-steddy proposed version.
![image](https://user-images.githubusercontent.com/18567511/188860095-f953b69c-49c1-4ea1-93cb-f209bb14dc7d.png)

**Comment by ivanromakh** [comment]
mobile version.
![image](https://user-images.githubusercontent.com/18567511/188863870-5321880e-f16d-4331-81f0-dce4c986ecb8.png)

**Comment by s-steddy** [comment]
Page is looking really good, here are hopefully the final three points:

- [ ] I misspelled welcome as wellcome. Correct header in popup to Welcome to Brillder for Libraries. Also remove full stop (dot) at end of line 
- [ ] If user expands view, they can see Login | Register. Ensure these have separate links to relevant pages, and that ?origin=library appended to url still means they'll be redirected to brick after going through process.
- [ ] remove ?referalid= part of generated link for sharing the brick, so we just have ?origin= part. Don't want to make librarians'/teachers' names apparent
